pennylane_qulacs/qulacs_device.py

```diff
@@ -26,11 +56,16 @@
 Z = np.array([[1, 0], [0, -1]])
 H = np.array([[1, 1], [1, -1]])/np.sqrt(2)
 SWAP = np.array([[1, 0, 0, 0], [0, 0, 1, 0], [0, 1, 0, 0], [0, 0, 0, 1]])
-CSWAP = block_diag(I, I, SWAP)
 
 rx = lambda theta: np.cos(theta / 2) * I + 1j * np.sin(-theta / 2) * X
 ry = lambda theta: np.cos(theta / 2) * I + 1j * np.sin(-theta / 2) * Y
 rz = lambda theta: np.cos(theta / 2) * I + 1j * np.sin(-theta / 2) * Z
+phase_shift = lambda phi: np.array(
```

Comment:
(Out of scope): could move these to use the more performant `math` and `cmath` variants as a future addition (as was done in the https://github.com/XanaduAI/pennylane/pull/567 PR)

Reply:
Oh, that might actually be a good idea to do right away. :thought_balloon: